code/modules/research/xenobiology/xenobio_camera.dm

```diff
@@ -0,0 +1,163 @@
+//Xenobio control console
+/mob/camera/aiEye/remote/xenobio
+	visible_icon = 1
+	icon = 'icons/obj/abductor.dmi'
+	icon_state = "camera_target"
+
+
+/mob/camera/aiEye/remote/xenobio/setLoc(var/t)
+	var/area/new_area = get_area(t)
+	if(new_area.name && new_area.name == "Xenobiology Lab")
```

Comment:
Then the console is unusable until you fix it.

I guess I could add an or to check for the Xenobio area type so it's always usable there no matter what.